### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by matsduf**
Removed detailed OS version information plus added lang da. Solves issue #539.

### CODE REVIEW

**Comment by sandoche2k** [comment]
@matsduf Looks Ok for me.

**Comment by matsduf** [comment]
@sandoche2k, I take your comment as an approval.

File: README.md [hunk 1]
```diff
@@ -60,15 +60,15 @@ before we can include support for Debian 9._
...
+Debian 7         | 5.5   |   (9.3)
+Debian 8         | 5.5   |   (9.4)
 FreeBSD 10.3     | (5.6) |   9.5       
 FreeBSD 11.1     | (5.6) |   9.5      
 Ubuntu 14.04     | 5.5   |   9.3
 Ubuntu 16.04     | 5.7   |   (9.5)
 
-_PostgreSQL has not been tested on CentOS and Debian 7.11. We had issues
-on Debian 8.9 and Ubuntu 16.04 that we could not resolve before release._
+_PostgreSQL has not been tested on neither CentOS 7 nor Debian 7. We had issues
```
Comment (mattias-p) [inline README.md]: Remove "not"?

File: README.md [hunk 2]
```diff
@@ -91,6 +91,9 @@ Zonemaster comes with localization for these locales:
 * en.UTF-8
 * fr.UTF-8
 * sv.UTF-8
+* da.UTF-8 (*)
+
+*) All strings in the Danish translation are not yet translated.
```
Comment (mattias-p) [inline README.md]: "Some strings have not yet been translated to Danish."?

File: README.md (hunk 1 above)
Comment (matsduf) [reply README.md]: I'll rewrite to something simpler.

**Comment by mattias-p** [comment]
This is related to #540, right?

**Comment by matsduf** [comment]
Correct. Solves issue #539.

**Pull request merged**